Dumper/HtmlDumper.php

```diff
@@ -624,6 +624,9 @@ function showCurrent(state)
 
 })(document);
 </script><style>
+:root {
+    color-scheme: light dark;
```

Comment:
Yeah, that's true. Hmm it could be annoying for many devs to dump code into a view and have the color scheme suddenly change. But on the flip side, when devs are dumping code, they're obviously debugging, so they probably care more about what they see inside the dumper than how the UI looks like, don't you think? 😅

Personally, I'd choose the second option, but yeah I understand the first one, which makes more sense in an objective point of view.

That said, I'm not sure if I should submit the PR on the mono-repo you, as you say [here](https://github.com/symfony/var-dumper/pull/30#issuecomment-2009343889), should I?